app/assets/stylesheets/administrate/base/_forms.scss

```diff
@@ -1,78 +1,99 @@
 fieldset {
-  background: lighten($base-border-color, 10);
-  border: $base-border;
-  margin: 0 0 ($base-spacing / 2) 0;
-  padding: $base-spacing;
+  background-color: transparent;
+  border: 0;
+  margin: 0;
+  padding: 0;
 }
 
-input,
-label,
-select {
-  display: block;
-  font-family: $form-font-family;
-  font-size: $form-font-size;
+legend {
+  font-weight: 600;
+  margin-bottom: $small-spacing / 2;
+  padding: 0;
 }
 
 label {
-  font-weight: bold;
-  margin-bottom: $base-spacing / 4;
-
-  &.required:after {
-    content: "*";
-  }
+  display: block;
+  font-weight: 600;
+  margin-bottom: $small-spacing / 2;
+}
 
-  abbr {
-    display: none;
-  }
+input,
+select {
+  display: block;
+  font-family: $base-font-family;
+  font-size: $base-font-size;
 }
 
+[type="color"],
+[type="date"],
+[type="datetime"],
+[type="datetime-local"],
+[type="email"],
+[type="month"],
+[type="number"],
+[type="password"],
+[type="search"],
+[type="tel"],
+[type="text"],
+[type="time"],
+[type="url"],
+[type="week"],
+input:not([type]),
 textarea,
-#{$all-text-inputs},
-select {
-  background-color: white;
-  border-radius: $form-border-radius;
-  border: 1px solid $form-border-color;
-  box-shadow: $form-box-shadow;
+select[multiple] {
+  background-color: $white;
+  border: $base-border;
+  border-radius: $base-border-radius;
+  box-shadow: none;
   box-sizing: border-box;
-  font-family: $form-font-family;
-  font-size: $form-font-size;
+  font-family: $base-font-family;
+  font-size: $base-font-size;
   margin-bottom: $small-spacing;
-  padding: $form-input-padding;
-  transition: border-color;
+  padding: $base-spacing / 3;
```

Comment:
`$base-spacing / 3` should be written without spaces around the operator: `$base-spacing/3`